### ISSUE TITLE
Add a versioning system

### ISSUE THREAD

**Issue by bearbin** [cuberite/plugin-repository#24]

**Comment by tigerw** [comment]
There is one? It uses GitHub versions.

**Comment by bearbin** [comment]
Can you link an example?

**Comment by tigerw** [comment]
Consider my extremely function [Factions plugin](http://cuberiteplugins.azurewebsites.net/showplugin.php?id=46669268).

**Comment by bearbin** [comment]
But can you download the different versions easily from the plugin repo?

**Comment by tigerw** [comment]
Why would you want to! Latest is greatest :)

**Comment by sphinxc0re** [comment]
I agree with @bearbin here. We should offer the user commitwise version selection or selection by tag name/ release name
